flowauth/backend/flowauth/admin.py

```diff
@@ -52,12 +52,22 @@ def list_all_capabilities():
     -----
     Responds with {<capability_name>: {"id":<capability_id>, "permissions":{<right>:False}}}
     """
+    default_capacity_setting = current_app.config["DEMO_MODE"]
```

Comment:
Minor suggestion: just to keep the terminology consistent, how about renaming this to `default_permission` (or `default_permission_setting`), because the JSON key is also called `"permissions"`? Or is "capacity" used elsewhere in the code?

Reply:
Capability is used elsewhere (e.g. `flow` is a capability), but this would be better named `default_permission_setting` 👍